src/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './KeyringClient';
 export * from './rpc-handler';
 export * from './KeyringSnapControllerClient';
 export * from './KeyringSnapRpcClient';
+export * from './internal';
```

Comment:
Is it right to expose a submodule `internal` at this level? My impression is that it should only be used inside this package just from the name

Reply:
Solved in an internal conversation